build-tools/makefile_components/base_container_arm64.mak

```diff
@@ -0,0 +1,28 @@
+# Container section of standard makefile (arm64)
+##### PLEASE DO NOT CHANGE THIS FILE #####
+##### If one of these sections does not meet your needs, consider copying its
+##### contents into ../Makefile and commenting out the include and adding a
+##### comment about what you did and why.
+
+SANITIZED_DOCKER_REPO = $(subst /,_,$(DOCKER_REPO))
+
+DOTFILE_IMAGE = $(subst /,_,$(IMAGE))-$(VERSION)
+
+container-arm64: .container-arm64-$(DOTFILE_IMAGE) container-arm64-name
+
+# In CI environments, use the plain Docker build progress to not overload the CI logs
+PROGRESS := $(if $(CI),plain,auto)
+
+.container-arm64-$(DOTFILE_IMAGE): $(wildcard Dockerfile Dockerfile.in) container-arm64-name
+    # UPSTREAM_REPO in the Dockerfile.in will be changed to the value from Makefile; this is deprecated.
+    # There's no reason not to just use Dockerfile now.
+	@if [ -f Dockerfile.in ]; then sed -e 's|UPSTREAM_REPO|$(UPSTREAM_REPO)|g' Dockerfile.in > .dockerfile; else cp Dockerfile .dockerfile; fi
```

Comment:
This is the old stuff pre-multistage-build, so can be dropped out (or not if you don't want to)